Document PHP "all experimental features" (#21124)
DataDog/documentation: content/en/tracing/trace_collection/library_config/php.md

Enable the allocation size and allocation bytes profile type. Added in version `0.88.0`. When an active JIT is detected, allocation profiling is turned off for PHP version `8.0.0`-`8.1.20` and `8.2.0`-`8.2.7` due to a limitation of the ZendEngine.<br>
**Note**: This supersedes the `DD_PROFILING_EXPERIMENTAL_ALLOCATION_ENABLED` environment variable (`datadog.profiling.experimental_allocation_enabled` INI setting), which was available since `0.84`. If both are set, this one takes precedence.

`DD_PROFILING_EXPERIMENTAL_FEATURES_ENABLED`
: **INI**: `datadog.profiling.experimental_features_enabled`. INI available since `0.96.0`.<br>
**Default**: `0`<br>
Enable all experimental features.<br>

> **Note**: This setting overrides the more specific configurations and if enabled, toggling other experimental configuration settings will _not_ have an effect anymore.

Reviewer: ```suggestion
**Note**: This setting overrides the more specific configurations and if enabled, toggling other experimental configuration settings won't have an effect.
```
